src/functions.php

```diff
@@ -1,21 +1,22 @@
 <?php
 
-function produce_HTML_Head(): string {
-	return '<html lang="en-gb"><head><meta charset="utf-8"><title>Cosmic Collector</title>
-	<link rel="stylesheet" type="text/css" href="normalize.css">
-	<link rel="stylesheet" type="text/css" href="cosmic_collector.css"></head>';
-}
-
-
-function display_Cosmic_Event(array $db_row): string {
-	return '<div class="cosmic_event"><h2>' . $db_row['name'] . '</h2><h3>Caused by: ' . $db_row['caused_by']
-		. '</h3><h3>Creates: ' . $db_row['creates_effect'] . '</h3><h3>Produces: </h3></div>';
+function format_event_elements(array $elements): array {
+	$elements_produced = [];
+	foreach ($elements as $element){
+		if (array_key_exists('name', $element)) $elements_produced[] = $element['name'];
```

Comment:
even though technically you can put an if statement on one line, it's best to have it on multiple lines. see https://www.php-fig.org/psr/psr-12/